src/main/java/org/broadinstitute/hellbender/utils/FisherExactTest.java

```diff
@@ -37,7 +38,7 @@ public static double twoSidedPValue(final int[][] normalizedTable) {
 
         final AbstractIntegerDistribution dist = new HypergeometricDistribution(null, m+n, m, k);
         final double[] logds = support.mapToDouble(dist::logProbability);
-        final double threshold = logds[x[0][0] - lo];
+        final double threshold = logds[x[0][0] - lo] * REL_ERR;
```

Comment:
So I have no idea if this is mathematically supportable, but if it makes us match with R, that sounds good...